Fill out the Contributor Video Proposal with a draft. (#79)
InnerSourceCommons/InnerSourceLearningPath: contributor/video-proposal.md

## Description & Learning Promise

### What is this video about, and why is it important?

> The Contributor role is one the most central roles in InnerSource - without Contributors, there would not be anything to receive or discuss about. 

Reviewer: ```suggestion
The Contributor role is one the most central roles in InnerSource - without Contributors, there would not be anything to receive or discuss. 
```

> There's far more to it than just dumping some stuff in a branch or issue.

Reviewer: ```suggestion
There's far more to it than just dumping some code in a branch.
```

Author: In the outline we do highlight that no contribution is too small, especially contributions to issues etc. Thus a contribution does not always have to be code, although often it will likely be, which is why I added it to that sentence - I'm very open for something else than "stuff".

Reviewer: > very open for something else than "stuff"

content?

> This segment teaches allows you to learn about the mindset, the habits, technical approaches and other details around this role enabling you to successfully contribute to the host teams work, foster good relationships with them and hopefully be a happy InnerSource contributor.

Reviewer: ```suggestion
This section teaches the mindset, the habits, technical approaches and other details around this role.  It enables you to successfully contribute to the host teams work, foster good relationships and be a happy InnerSource contributor.
```

Author: Thanks for catching the defect sentence and splitting it up into two sentences! :)

### What you’ll learn—and how you can apply it

> * Introduction: How to contribute to an InnerSource project in a good and what it means to be an InnerSource Contributor  

Reviewer: ```suggestion
* Introduction: What is an InnerSource Contributor and why is it important.
```

Author: One more defect sentence found&fixed, thanks!

> * Becoming an InnerSource Contributor: Mindsets and habits of successful contributors

Reviewer: ```suggestion
* Becoming an InnerSource Contributor: Mindsets and habits that lead to becoming a contributor
```

Author: :+1:

> * The Contributor Ethos: Getting more specific on execution of the mindset and habits   

Reviewer: ```suggestion
* The Contributor Ethos: Mindset to keep while acting as an InnerSource Contributor.   
```

Reviewer: What do you think about this suggestion?

Reviewer: Great!

* Mechanics of Contributing: More technical aspects of contribution to ensure success
* Benefits of Contribution: What's in it for me (the guest), the receiving team (the host) and our company?
* Conclusion: Recap of this section and pointers to more content

### This video is for you because…

* You are a possible future Contributor to an InnerSource project and want to know what to expect 
* You are a host team member looking to open your work for outside contributors / inner sourcing and want to be prepared to know what being a contributor may be like 
* You are a product owner looking to support your TC or a (line) manager and want to know how the inner source approach is meant to work.

### Prerequisites
 
* Please watch the [Intro to InnerSource Video](https://www.safaribooksonline.com/videos/introduction-to-innersource/9781492041504)

> * Basic Understanding of the Trusted Committer role

Reviewer: Is there a specific video to watch here, perhaps?

Reviewer: [Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925)?

Reviewer: ```suggestion
* Also watch the [Introduction](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323925) to the Trusted Committer role.
```

> * The [lowering the barriers to entry](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323929) video of the Trusted Committer role can be a good addition to understand more of why the Trusted Committer might ask for different things.

Reviewer: ```suggestion
* The [Lowering the Barriers to Entry](https://learning.oreilly.com/videos/the-trusted-committer/9781492047599/9781492047599-video323929) video of the Trusted Committer role can be a good addition to understand more of why the Trusted Committer might ask for different things.
```

### Materials or downloads needed in advance
 
* none

### Audience level
_Describe the level of difficulty for the material, so that we can segment the intended audience_

> * _We should discuss this. PO had "familiar with the Agile or Lean process" here, I'm not sure if this part needs that._  

Reviewer: ```suggestion
* Familiar with Git pull request workflow.
```

Author: Yes, very good point that I can confirm. Do we have some material we want to point people to? Maybe something from GitHub or GitLab?

Reviewer: > some material we want to point people to? Maybe something from GitHub or GitLab?
Sounds good - I haven't looked to see what's out there, though.
